lib/charCounter.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+/* eslint-disable */
+
+/**
+ * Function to count how many characters are remaining in text fields
+ *
+ * @param {Object} input - inputs in this plugin's instance and their values as {String}
+ * *
+ * @module charCounter
+ */
+
+module.exports = function charCounter(input) {
+  var msg = input.parent.querySelector('.input-plugin-textarea'),
```

Comment:
The structure for `inputs`, as discussed is:

```yaml
inputs:
  target:
  all:
  parent:
```

This should be `inputs.target`. Don't query the class directly; this may be repeatable.